src/main/java/domain/GamePlayer.java

```diff
@@ -0,0 +1,48 @@
+package domain;
+
+import exception.DuplicateNumberException;
+import exception.InvalidNumberRangeException;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Stream;
+
+public class GamePlayer {
+
+    public static final String VALID_NUMBERS_RANGE = "[1-9]{3}";
+    public static final int VALID_NUMBERS_LENGTH = 3;
+    public static final String SPLIT_REGEX = "";
+    private final List<Integer> baseballNumbers;
```

Comment:
상수와 필드 사이에 한 칸 공백을 주면 읽기 수월할 것 같아요~

```suggestion
    public static final String VALID_NUMBERS_RANGE = "[1-9]{3}";
    public static final int VALID_NUMBERS_LENGTH = 3;
    public static final String SPLIT_REGEX = "";
    
    private final List<Integer> baseballNumbers;
```

Reply:
컨벤션에 더욱 주의하겠습니다!